test/e2e/Dockerfile.e2e

```diff
@@ -0,0 +1,25 @@
+FROM registry.suse.com/bci/golang:1.18 AS build
```

Comment:
According to what we have in `go.mod` I think this should be `registry.suse.com/bci/golang:1.19`